hadoop-ozone/common/pom.xml

```diff
@@ -30,6 +30,21 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
 
   <dependencies>
 
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-netty</artifactId>
+      <version>${io.grpc.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http2</artifactId>
+      <version>${netty.version}</version>
```

Comment:
With `netty-bom` imported in root `pom.xml`, `netty.version` does not need to be specified anywhere else.

Reply:
Reflected in latest commit, removed unnecessary version declarations in pom files.  Thanks @adoroszlai.